src/runtime/server/api/_hub/database/[command].post.ts

```diff
@@ -20,42 +20,35 @@ export default eventHandler(async (event) => {
   const db = hubDatabase()
 
   if (command === 'exec') {
-    const { query } = await readValidatedBody(event, z.object({
-      query: z.string().min(1).max(1e6).trim()
-    }).parse)
+    const { query } = await readValidatedBody(event, statementValidation.pick({ query: true }).parse)
     return db.exec(query)
   }
   if (command === 'dump') {
     return db.dump()
   }
   if (command === 'first') {
-    const { query, params, colName } = await readValidatedBody(event, z.object({
-      query: z.string().min(1).max(1e6).trim(),
-      params: z.any().array(),
-      colName: z.string().optional()
-    }).parse)
-    if (colName) {
-      return db.prepare(query).bind(...params).first(colName)
+    const { query, params, columnName } = await readValidatedBody(event, z.intersection(
+      statementValidation,
+      z.object({ columnName: z.string().optional() })
```

Comment:
Changing `colName` to `columnName` will be a breaking change for all users.
`colName` is a common name for column fields in both `db0` and `drizzle`, so it is good to keep it as is

Reply:
Ok. It is weird that it is `colName`(singular) and `columnsNames`(plural) though